scripts/the-pied-pine-perf.yml

```diff
@@ -1,6 +1,6 @@
 config:
-  target: http://localhost:3001
-  timeout: 120
+  target: http://localhost:3000
```

Comment:
will this interfere with the redirect uri callback when getting the access token in teardownBackend()?

Reply:
Good catch - missed that in conflict resolution.

Reply:
Actually I don't think so.

Reply:
I'll change it back because I didn't intend to change it, but really this doesn't factor in unless we don't specify the root of baseUrl or in your case orchestratorBaseUrl